### PR TITLE
docs: Connector Builder contribution flow text-based docs

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

Another little cleanup of contributing via Builder! This adds contrib flow docs into the "No-Code" section. 

## TODO

To follow up, we need to:
- Improve low-code docs
- Add an arcade video on how to contribute via builder, or a loom.

I'm working on both of those. Just wanted to start shipping pieces out.

### CODE REVIEW

File: docs/connector-development/schema-reference.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Schema Reference
 
+:::note
+You only need this if you're building a connector with Python or Java CDKs.
```
Comment (girarda) [inline docs/connector-development/schema-reference.md:4]: probably good to point to the analogous page for builder and low-code. I think the closest topic is https://docs.airbyte.com/connector-development/connector-builder-ui/record-processing#declared-schema

File: docs/connector-development/connector-builder-ui/overview.md [hunk 2]
```diff
@@ -1,56 +1,41 @@
...
+If you'd like to share your connector with other Airbyte users, you can contribute it to Airbyte's GitHub repository right from the Builder.
 
-From this point on your connector is a regular low-code CDK connector. It can now be distributed as a docker image and be made part of the regular Airbyte connector catalog. For more information, read the [overview page for the publishing process](/connector-development/#publishing-a-connector).
+1. Click "Publish" chevron -> "Contribute to Marketplace"
+2. Fill out the form: add the connector description, and provide your GitHub PAT (Personal Access Token) to create a pull request
+3. Click "Contribute" to submit the connector to the Airbyte catalog
 
-:::note
-Connector Builder UI is in beta, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io
+Reviews typically take under a week.
```
Comment (girarda) [inline docs/connector-development/connector-builder-ui/overview.md:39]: are we committing to this timeline?

**Review by girarda — APPROVED** [review]

File: docs/connector-development/connector-builder-ui/overview.md (hunk 2 above)
Comment (natikgadzhi) [reply docs/connector-development/connector-builder-ui/overview.md:39]: Not entirely true — aspirational, but I would be happy if anyone asks me and cites this doc.

File: docs/connector-development/schema-reference.md (hunk 1 above)
Comment (natikgadzhi) [inline docs/connector-development/schema-reference.md:4]:
```suggestion
You only need this if you're building a connector with Python or Java CDKs.
If you're using Connector Builder, you can use [declared schemas](./connector-builder-ui/record-processing#declared-schema) instead.
```

**Comment by natikgadzhi** [comment]
More to do, but this one will automerge unless I screwed up the linkst

**Pull request merged**